src/modules/visitor-space/components/CreatorFilterForm/CreatorFilterForm.tsx

```diff
@@ -80,6 +80,7 @@ const CreatorFilterForm: FC<CreatorFilterFormProps> = ({ children, className })
 					)}
 
 					<CheckboxList
+						className="c-checkbox-list"
```

Comment:
Classname is already available since it's added in the [react-components component](https://github.com/viaacode/react-components/pull/81/files#diff-9d9c18b916cae7b47463b8d0d45de4a41d518fe9081daa35a284477cfcb3c48bR18)